src/styles/button.scss

```diff
@@ -0,0 +1,28 @@
+.comp__to-buy {
+  display: block;
+  width: 166px;
+  border: 1px solid $blue_color;
+  margin-top: 16px;
+  margin-left: 16px;
+  background-color: $blue_color;
+  color: $white_color;
+  text-align: center;
+  line-height: 40px;
+  height: 40px;
+  border-radius: $border-radius;
+  font-weight: 700;
+  font-size: 14px;
+  font-family: $font-family;
```

Comment:
Ensure that the variables `$blue_color`, `$white_color`, `$border-radius`, and `$font-family` are defined in a separate SCSS variables file. This is important for maintaining consistency across your styles and making updates easier.